tests/e2e/tests/web/critical-path/rdi/add-job.e2e.ts

```diff
@@ -0,0 +1,80 @@
+import { t } from 'testcafe';
+import { RdiInstancePage } from '../../../../pageObjects/rdi-instance-page';
+import { AddNewRdiParameters, RdiApiRequests } from '../../../../helpers/api/api-rdi';
+import { commonUrl } from '../../../../helpers/conf';
+import { MyRedisDatabasePage } from '../../../../pageObjects';
+import { RedisOverviewPage } from '../../../../helpers/constants';
+import { RdiInstancesListPage } from '../../../../pageObjects/rdi-instances-list-page';
+import { BrowserActions } from '../../../../common-actions/browser-actions';
+
+const rdiInstancePage = new RdiInstancePage();
+const rdiApiRequests = new RdiApiRequests();
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const rdiInstancesListPage = new RdiInstancesListPage();
+const browserActions = new BrowserActions();
+
+const rdiInstance: AddNewRdiParameters = {
+    name: 'testInstance',
+    url: 'http://localhost:4000',
+    username: 'username',
+    password: 'password'
+};
+
+//skip the tests until rdi integration is added
+fixture `Add job`
+    .meta({ type: 'critical_path' })
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await t.maximizeWindow();
+        await myRedisDatabasePage.setActivePage(RedisOverviewPage.Rdi);
+        await rdiApiRequests.addNewRdiApi(rdiInstance);
+        await rdiInstancesListPage.reloadPage();
+        await rdiInstancesListPage.clickRdiByName(rdiInstance.name);
+
+    })
+    .afterEach(async() => {
+        await rdiApiRequests.deleteAllRdiApi();
+    });
+test('Verify that user can add, edit and delete job', async() => {
```

Comment:
Please add it to checklist

Reply:
I will do it later
I have not added the story to checklist to not forget merge dev pull request to common branch